### ISSUE TITLE
Feature Request

### ISSUE THREAD

**Issue by bryanhorner** [fluid-player/fluid-player#583]
Hi,

Not sure if this has been asked before or is impossible but it would be great if Fluidplayer could pull in and play YouTube and Vimeo videos. 

Thanks!

**Comment by AjmalPraveeN** [comment]
Yes, it will be very useful. For ex: "plyr" has the option if you use youtube link in the src, it automatically loads the video and plays as native player.
Fluid player too need such improvement to support youtube videos 👍 hope dev will look into this asap.

**Comment by leroybm** [comment]
Create a new internal ticket to track this, we should implement something close to what Plyr has
